lang/semgrep-grammars/src/semgrep-julia/grammar.js

```diff
@@ -13,6 +13,10 @@ module.exports = grammar(base_grammar, {
     [$.typed_parameter, $._expression],
   ]),
 
+  externals: ($, previous) => previous.concat([
+    $._semgrep_metavariable
```

Comment:
Why this has to be an external symbol? Looking at the C code, it looks like /$[A-Z][a-zA-Z0-9]*/  would also do the job.
Can you add a comment why semgrep_metavariable can't be defined in grammar.js?

Reply:
You were actually completely right. The reason I shyed away from a regex is that, in my initial attempt to support identifiers as metavariables using a regex, this caused us to fail situations like `$Pack`. I believe the reason for this was that I was using the `choice` and regex _inside of_ a `token`, because I hadn't yet made the change to no longer make `identifier` the word token.

It turns out, when we make it a choice of the regex at the nonterminal level, the tests pass and we go through. So this PR can be made much simpler.